addons/im_livechat/controllers/chatbot.py

```diff
@@ -67,6 +67,9 @@ def chatbot_trigger_step(self, channel_id, chatbot_script_id=None):
             if chatbot.exists():
                 next_step = chatbot.script_step_ids[:1]
 
+        is_last = next_step._is_last_step(discuss_channel) if next_step else None
+        if not discuss_channel.livechat_outcome and (not next_step or is_last):
```

Comment:
Shouldn't it specifically be in `if not next_step:`?
As this is the only case where we set livechat_active = False.